docs/source/workbench/ipythonconsole.rst

```diff
@@ -0,0 +1,13 @@
+.. _WorkbenchIPythonConsole:
+
+===============
+IPython Console
+===============
+
+.. image:: ../images/Workbench/IPythonWidget.png
+
+This is a python interpreter which has access to the Mantid API and can
+interface with the rest of the Workbench. Through the interpreter it is
+possible to interact with Workbench's algorithms, workspaces and plots. It
+allows you to manipulate things much like the script window can, but in an
+interpreter style.
```

Comment:
Can we mentino that this executes python automatically after return is pressed, and perhaps include a link to ipython/jupyter?